README.md

```diff
@@ -9,27 +10,66 @@ PreguntonicBackend es el núcleo inteligente y robusto que impulsa la experienci
 2. maven
 3. docker
 
-### Run service in local 
+### Run Locally 
 
-Run as a regular Java Spring Boot application.
+#### With Maven ☕
 
-#### With Maven
+Create `.jar` file
 
 ```bash
 mvn clean package
+```
+
+Run as a service
+
+```bash
 java -jar target/preguntonic-backend-0.0.1-SNAPSHOT.jar
 ```
 
-### Run service in local using Docker
-using docker 
-1. create a .jar file
-2. build the docker image
-    ```bash
-    docker compose build preguntonic-backend
-    ```
-3. run the docker image
-    ```bash
-    docker compose up
-    ```
-4. open the browser and go to http://localhost:8080
+### With Docker 🐳
+
+Create `.jar` file
+
+```bash
+mvn clean package
+```
```

Comment:
With new docker this step is not necessary but this is still nedded:

```
2. build the docker image
    ```bash
    docker compose build preguntonic-backend
    ```
```